cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -200,7 +207,36 @@ friendlyTxBodyImpl
                   , "update proposal" .= friendlyUpdateProposal txUpdateProposal
                   , "validity range" .= friendlyValidityRange era (txValidityLowerBound, txValidityUpperBound)
                   , "withdrawals" .= friendlyWithdrawals txWithdrawals
-                  ])
+                  ]) ++
+                (inEonForEra
```

Comment:
Rather than using `++` it's ok to return `"governance actions" .= Null`. Although if you really don't like that you can use a combination of `maybe` and `mempty` to get the desired result.

Reply:
I wouldn't have used `Null` myself directly, but since this is the behavior of many existing fields; I did this change to be consistent :+1: (and code is slightly nicer to read)